programs/cleos/main.cpp

```diff
@@ -503,8 +505,19 @@ using std::cout;
 void send_actions(std::vector<chain::action>&& actions, packed_transaction::compression_type compression = packed_transaction::none ) {
    auto result = push_actions( move(actions), compression);
 
+   string jsonstr;
+   if (tx_json_save_file.length()) {
+      if (jsonstr.length() == 0) {
+         jsonstr = fc::json::to_pretty_string( result );
+      }
+      std::ofstream out(tx_json_save_file);
```

Comment:
Should capture and report a better error message indicating the file that is unable to open.